client-ng1/src/app/assets/i18n/us_US.json

```diff
@@ -168,6 +168,61 @@
       "explanation": "An error occured. Please try again and if the error is still there, do not hesitate to contact us in order to allow us to correct it"
     }
   },
+  "account": {
+    "creation": {
+      "title": "Account creation",
+      "description": "To create an account, we need to know some information that will allow us to contact you. The minimum information includes an email to confirm, a login ID and your full name. Your email address will never be disclosed to third parties.",
+      "descriptionsocial": "To create an account, we need to know some information that will allow us to contact you. Your email address will never be disclosed to third parties.",
```

Comment:
...know some informationS ? 
To create an account, we ask you to provide information that will enable to enhance your site visit and help us to contact you?
Mais je ne suis pas un pro de l'anglais